### ISSUE TITLE
Generated tess_version.h

### ISSUE THREAD

**Issue by amitdo** [tesseract-ocr/tesseract#1515]
When working on a topic branch, and compiling, git say: 

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	api/tess_version.h

nothing added to commit but untracked files present (use "git add" to track)
```
@stweil, I think `tess_version.h` should be added to `.gitignore`.

**Comment by stweil** [comment]
This was already on my personal work list. It is now addressed by pull request #1517.

It's possible to avoid a mix of source files and generated files by running configure and make from some special build directory instead of running it from the Tesseract root directory.

**Comment by amitdo** [comment]
>It's possible to avoid a mix of source files and generated files by running configure and make from some special build directory instead of running it from the Tesseract root directory.

Yes, I **usually** build inside 'build' directory (which is listed in .gitignore).
